src/clib/xtg/cube_export_rmsregular.c

```diff
@@ -89,8 +89,11 @@ cube_export_rmsregular(int nx,
             for (i = 1; i <= nx; i++) {
 
                 ic = x_ijk2ic(i, j, k, nx, ny, nz, 0);
-
-                value = cubevalsv[ic];
+                if (ic < 0) {
+                    value = UNDEF_CUBE_RMS;
```

Comment:
This seems to be a common way to handle this case, should I perhaps adopt that everywhere instead of `throw_exception`? There is a risk of missing some bugs though.

Reply:
I would also consider that we could end up changing the expected behaviour.

Consider (here the `throw_exception` is added in this PR):

```
if (ib < 0 || ic < 0) {
    throw_exception("Outside boundary");
    return EXIT_FAILURE;
}
if (option1 == 1 && actnumsv[ib] == 0) {
    dx[ic] = UNDEF;
    dy[ic] = UNDEF;
    continue;
}
```

If we were to simply adopt the `UNDEF` behaviour, the function would treat invalid lookups as it treats in-active cells. The former should be an error while the latter is pretty much a valid setup, agree?

I agree that the example you gave though is more in the line of an error.

Or at least, it's a bit weird that both scenarios in my example would provide the same result.

Reply:
So in commit 55bd771d252c2e059e745f7cd9fb803e46070dd6 you're going for the `UNDEF` value for both `cube_export_rmsregular` and `cube_export_segy`, but in https://github.com/equinor/xtgeo/pull/552/commits/34d5c158bcb2d47a1349ac588e065b38065f7020 changes the `cube_export_rmsregular` back to throwing, while `cube_export_segy` remains the same?